### DISCUSSION TITLE
Import over the gui does not start

### DISCUSSION THREAD

**Discussion by DavidePrincipi** [jsuto/piler#58] [Q&A] [answered]
I'm running the Piler container version 1.4.4, in a development environment.

I enabled the [Import over the gui](https://mailpiler.com/import-over-the-gui/) feature, and added a job after testing the credentials (all ok).

After an hour, the job is still in "Pending" state: 

![image](https://github.com/jsuto/piler/assets/2920838/dcfe4cd5-5497-4755-9fc4-9a516594bf82)

I expected a cron task defined under /etc/cron*, but I can't find anything related to it inside the container image.

Thanks for any help!

**Comment by DavidePrincipi** [comment]
Thanks to Steph's suggestion, I looked at `/usr/share/piler/piler.cron`: 

```
### PILERSTART
5,35 * * * * /usr/libexec/piler/indexer.delta.sh
30   2 * * * /usr/libexec/piler/indexer.main.sh
40 3 * * * /usr/libexec/piler/purge.sh
*/15 * * * * /usr/bin/indexer --config /etc/piler/manticore.conf --quiet tag1 --rotate
*/15 * * * * /usr/bin/indexer --config /etc/piler/manticore.conf --quiet note1 --rotate
*/5 * * * *  /usr/bin/find /var/piler/www/tmp -type f -name i.\* -exec rm -f {} \;
*/5 * * * * /usr/bin/find /var/piler/error -type f|wc -l > /var/piler/stat/error

### optional
###30 6 * * * /usr/bin/php /usr/libexec/piler/generate_stats.php --webui /var/piler/www >/dev/null
###*/5 * * * * /usr/libexec/piler/import.sh

### PILEREND
```

The `import.sh` script is commented out. Running it manually does what I want

![image](https://github.com/jsuto/piler/assets/2920838/c3e61812-e363-453d-ba2e-a3152d8f906e)

**Comment by jsuto** [comment, marked as answer]
"crontab -l -u piler" show the crontab entries. By default import.sh is not enabled. Uncomment it, and you should be fine.

**Reply by DavidePrincipi** [reply]
After the container is running, I try to edit it with sed:

    podman exec piler-app sed -i -r '/^###[3\*]/ s/^###//' /usr/share/piler/piler.cron

**Reply by jsuto** [reply]
Great, in the meantime I make this enabled by default for the next release.
